### ISSUE TITLE
Useless code in the SDN-DO

### ISSUE THREAD

**Issue by ivanocerrato** [netgroup-polito/frog4-sdn-do#19]
@gabrielecastellano , according to @farmanullahcs there is some code that is useless. 
Can he remove this code? Of course, he works in a branch; the merge into the master will be done only after that we checked that everything still works fine.

**Comment by frisso** [comment]
@ivanocerrato: which code are you referring to?

**Comment by farmanullahcs** [comment]
No usages found for this code,
1) https://github.com/netgroup-polito/frog4-sdn-do/blob/master/do_core/exception.py#L38-L111
2) https://github.com/netgroup-polito/frog4-sdn-do/blob/master/do_core/user_authentication.py#L74-L96

**Comment by ivanocerrato** [comment]
Moreover, is this code https://github.com/netgroup-polito/frog4-sdn-do/blob/master/do_core/exception.py#L122-L126 (that has not been linked by @farmanullahcs) used?
If yes, I would suggest to rename the class, since it refers to Heat, which is an OpenStack component (and has anything to do with the SDN-DO).

**Comment by frisso** [comment]
@gabrielecastellano: What do you think? Should we remove the above code?

**Comment by gabrielecastellano** [comment]
It should be ok if farman remove it in the new branch for the moment.
Then I will merge it into the master.

**Comment by farmanullahcs** [comment]
Useless code removed in the branch, [frog4-deployment](https://github.com/netgroup-polito/frog4-sdn-do/tree/frog4-deployment)
